gtk/src/inverted/install-inverted.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
Could we add a description of what "inverted" refers to? It's bit cryptic in the code

Reply:
Should I add this in description:

when `inverted` is true it will create symlinks like this:
- **Yaru-inverted** symlinked to **Yaru-dark** (*Yaru inverted is Dark-gtk and Light-shell*)
-  **Yaru-inverted-dark** symlinked to **Yaru** (*Yaru-inverted-dark is Light-gtk and Dark-shell*)

This two symlinks will be used if, the switch for ***keep gtk and shell in sync*** is disabled.

Reply:
That's fine for me. The main point was to describe the combination of shell and gtk used.